example/main.jsonnet

```diff
@@ -33,7 +33,7 @@ local dex = (import 'github.com/observatorium/deployments/components/dex.libsonn
   },
 };
 
-local obs = (import 'github.com/observatorium/deployments/environments/base/observatorium.jsonnet');
+local obs = (import '../environments/base/observatorium.jsonnet');
```

Comment:
how is this working?

Reply:
Yes, this needs to be fixed and we need to add a test case that checks if this compiles correctly.